### PR TITLE
Update chaos-engineering.md

### PR DESCRIPTION
**Pull request by gunnargrosch**
Added serverless functions under Faults to show that chaos engineering has benefits there as well.

### CODE REVIEW

**Comment by PRMerger7** [comment]
@gunnargrosch : Thanks for your contribution! The author(s) have been notified to review your proposed change.

File: docs/framework/resiliency/chaos-engineering.md [hunk 1]
```diff
@@ -148,8 +148,12 @@ Faults that can be leveraged to inject chaos.
 |  | Rack |  |
 |  | UPS  |  |
 |  | Datacenter |  |
+| **Functions** | Latency |  |
```
Comment (absheik) [inline docs/framework/resiliency/chaos-engineering.md:151]: @gunnargrosch can you move the functions to be right after Platform and before Application specific in the table?

**Comment by PRMerger7** [comment]
@gunnargrosch : Thanks for your contribution! The author(s) have been notified to review your proposed change.

File: docs/framework/resiliency/chaos-engineering.md (hunk 1 above)
Comment (gunnargrosch) [reply docs/framework/resiliency/chaos-engineering.md]: @absheik Moved it and commited.

**Comment by adamboeglin** [comment]
#sign-off

**Comment by PRMerger13** [comment]
@adamboeglin: I'm sorry - only the author of this article, @absheik, can sign off on your changes. But we do have an exception process - if you are on the Microsoft content or product team for this product area, you can ask the PR review team to review and merge it by sending mail to the techdocprs alias.

**Comment by ktoliver** [comment]
@adamboeglin @absheik - In line 148, if "blacklist" isn't used in the system UI, it should be changed to "safe list" or "allow list" per Microsoft style. Please advise. (I can make the edit before merging the PR.)

Thanks.

**Comment by PRMerger7** [comment]
@absheik : Thanks for your contribution! The author(s) have been notified to review your proposed change.

**Comment by absheik** [comment]
#sign-off

**Pull request merged**